getting_started/first_2d_game/07.finishing-up.rst

```diff
@@ -45,12 +45,15 @@ Since the game is played with keyboard controls, it would be convenient if we
 could also start the game by pressing a key on the keyboard. We can do this with
 the "Shortcut" property of the ``Button`` node.
 
+First, return to the *Project -> Project Settings* and click on the *Input Map*
+tab.  Add another action (like you did for the movement keys) called
+``start_game`` mapped to ``space``.
```

Comment:
I'm not fond of this one. To me, it makes sense to stick to the built-in ui_* events for UI actions. Here, we really want the `ui_select` mapping. The linked issue this PR tries to address is specifically about not using ui_* input actions for gameplay controls.

Reply:
Yeah, I can see that.  You really are doing a `ui_select` type action for the shortcut.  I can remove this change from the PR.  Also, the associated demo code change PR (https://github.com/godotengine/godot-demo-projects/pull/663) can then be closed with no merge, as the demo project already uses `move_*` actions for the player movement and `ui_select` for the start button shortcut.